client/configuration/webpack/obsoleteHTMLTemplate.html

```diff
@@ -38,27 +35,35 @@
       <a
         href="https://www.google.com/chrome/?hl=en%22"
         aria-label="Download Google Chrome"
+        target="_blank"
       >
-        <img
-          src="https://cellxgene.cziscience.com/s3/cellxgene/static/images/chrome.png"
-          style="width: 80px; height: 80px"
-        />
+        <img src="/images/chrome.png" style="width: 80px; height: 80px" />
```

Comment:
We'll need to see in `dev` if we need to remove the `/` prefix in `src`, since I think it was just for local testing and might break in deployed environment!